components/postmark/sources/new-inbound-email-received/new-inbound-email-received.mjs

```diff
@@ -0,0 +1,36 @@
+import postmark from "../../postmark.app.mjs";
+
+export default {
+  key: "postmark-new-inbound-email-received",
+  name: "New inbound email received",
+  description:
+    "Emit new event when an email is received by the Postmark server",
+  version: "0.0.1",
+  type: "source",
+  props: {
+    postmark,
+    http: {
+      type: "$.interface.http",
+      customResponse: true,
+    },
+  },
+  hooks: {
+    async activate() {
+      return this.postmark.setServerInfo({
+        InboundHookUrl: this.http.endpoint,
+      });
+    },
+    async deactivate() {
+      return this.postmark.setServerInfo({
+        InboundHookUrl: "",
+      });
+    },
+  },
+  async run(event) {
+    this.http.respond({
+      status: 200,
+    });
+
+    this.$emit(event);
```

Comment:
You need emit a event with a `id`, `summary` and `ts`.

Example: https://github.com/PipedreamHQ/pipedream/blob/aedf057dde1d963fd39fa7206c14314cc937602f/components/asana/sources/new-story/new-story.mjs#L49-L53

Reply:
Added event metadata as with the other source.